File: algebra/factorial/c#/factorial.cs
using System;

namespace Math {
    class Factorial {
        public static int factorial(int num) {
            int factorial = 1;

            for (int i=1;i<=num;i++) {
                factorial *= i;
            }
            return factorial;
        }

        public static void Main(string[] args) {
            int result = factorial(5);
            Console.WriteLine(result);
        }
    }
}

File: algebra/factorial/java/Factorial.java
// Program for finding factorial of a number

import java.util.Scanner;

public class Factorial {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter number : ");
		int n = sc.nextInt();
		int result = 1;
		for (int i=n; i>=1; i--) {
			result = result * i;
		}System.out.println("Your factorial is "+result);

	}

}
